draft-ietf-quic-transport.md

```diff
@@ -4055,9 +4053,11 @@ Final Offset:
 
 ## CONNECTION_CLOSE frame {#frame-connection-close}
 
-An endpoint sends a CONNECTION_CLOSE frame (type=0x02) to notify its peer that
-the connection is being closed.  CONNECTION_CLOSE is used to signal errors at
-the QUIC layer, or the absence of errors (with the NO_ERROR code).
+An endpoint sends a CONNECTION_CLOSE frame (type=0x02 or 0x03) to notify its
+peer that the connection is being closed.  The CONNECTION_CLOSE with a frame
+type of 0x02 is used to signal errors at the QUIC layer, or the absence of
+errors (with the NO_ERROR code).  The CONNECTION_CLOSE frame with a type of 0x03
+is used to signal an error with the protocol that uses QUIC.
```

Comment:
I suggest defing a subtype name such as "a CONNECTION_CLOSE of subtype TRANSPORT_CLOSE (0x02) or APPLICATION_CLOSE (0x03)". See my comment on http draft on using explicit frame code 0x03. This makes it difficult for external documents to discuss QUIC, especially across versions where the code might change but the semantics remain.